gCoffee/app/src/main/java/com/mborowiec/gcoffee/Order.java

```diff
@@ -106,10 +107,23 @@ private void makeOrder(String coffee) {
                 Toast.LENGTH_SHORT).show();
     }
 
+    /**
+     * Checks whether the menu contains the ordered item. If it does, asks to confirm order.
+     * @param data incoming deeplink
+     */
     private void handleDeepLinks(Uri data) {
-        String coffee;
-        coffee = data.getQueryParameter("inMenuName");
-        showConfirmationDialog(coffee);
+            String coffee;
```

Comment:
It might be a rendering problem, but it looks like the intention changed with this commit (thus, it was either wrong before the commit or is wrong now). It might be interesting so setup some checks for formatting and related issues. Would you be interested to look into this?

Reply:
As it took me a while to understand the comment: indentation :P